cypress/integration/helpcenter-panel.spec.ts

```diff
@@ -0,0 +1,74 @@
+describe("Helpcenter panel", () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:45000/iframe.html?id=core-helpcenter-panel--helpcenter-panel');
+  });
+
+  const transitionTime = 500;
+
+  const url = 'https://core.dso-toolkit.nl/_1376-helpcenter-panel/?path=/docs/helpcenter-panel--helpcenter-panel';
+
+  function prepareComponent() {
+    cy.get('dso-helpcenter-panel')
+      .should('have.attr', 'label', 'Hulp nodig')
+      .shadow()
+      .as('dsoHelpcenterPanel')
+      .find('div.wrapper div.iframe-container')
+      .as('iframeContainer')
+      .get('@dsoHelpcenterPanel')
+      .find('button.open-button')
+      .as('openButton')
+      .get('@dsoHelpcenterPanel')
+      .find('div.wrapper button.close-button')
+      .as('closeButton');
+  }
+
+  it('should be accessible', () => {
+    prepareComponent();
```

Comment:
Wanneer `prepareComponent` geen parameters heeft kunnen we dit beter kwijt in de `beforeEach()`

Reply:
Duidelijk